src/Internal/Wallet/Cip30.purs

```diff
@@ -98,7 +119,7 @@ mkCip30WalletAff
 mkCip30WalletAff walletName enableWallet = do
   wallet <- toAffE enableWallet
   -- Ensure the Nami wallet has collateral set up
-  whenM (isNothing <$> getCollateral wallet) do
+  whenM (isNothing <$> getCollateral wallet mempty) do
```

Comment:
A reminder that we should replace these memptys with 5ada. You could use `Ctl.Internal.BalanceTx.Collateral.Select.minRequiredCollateral` (except in places with import cycles or examples)